osquery/tables/system/darwin/secureboot.mm

```diff
@@ -176,17 +184,120 @@ Status getSecureBootSetting(Row& row) {
   return Status::success();
 }
 
+Status getAarch64SecureBootSetting(Row& r) {
+  @autoreleasepool {
+    // BEWARE: Because of the dynamic nature of the calls in this function, we
```

Comment:
This pattern for accessing SPSupport.framework is adapted from the connected_displays table. I think if another table is going to use this it should be refactored out into a helper function.